app/main.py

```diff
@@ -2,0 +1,38 @@
+import os
+
+
+def move_file(command: str) -> None:
+    components = command.split()
+    first_part, second_part, third_part = components
+    if len(components) == 3 and first_part == "mv":
+
+        file_name = second_part
+        path = third_part
+        dirs = third_part.split("/")
+
+        if len(dirs) > 1 and len(dirs) != 2:
+            current_path = dirs[0]
+            if not os.path.exists(current_path):
+                os.mkdir(current_path)
+
+            for folder in dirs[1:-1]:
+                if folder is not None:
+                    current_path = os.path.join(current_path, folder)
+                    if not os.path.exists(current_path):
+                        os.mkdir(current_path)
```

Comment:
Similar to the previous comment, ensure that you handle existing directories properly when creating nested directories. Use `os.makedirs(current_path, exist_ok=True)` to avoid errors if the directory already exists.